main.cpp

```diff
@@ -27,7 +27,7 @@ void PrintHelp() {
 }
 
 int main(int argc, char *argv[]) {
-	long long volumeSize = 0;
+	uint64_t volumeSize = 0;
```

Comment:
Is `size_t` even better?

Reply:
Yes, it is. I will move all the `Statistics` fields to `size_t`